internal/services/costmanagement/client/client.go

```diff
@@ -6,10 +6,10 @@ package client
 import (
 	"fmt"
 
-	"github.com/hashicorp/go-azure-sdk/resource-manager/costmanagement/2021-10-01/exports"
```

Comment:
I would probably keep the old version as well. I believe there are some gaps between what's supported in each version.

@jojoh would know more.

Reply:
If we have a provision to keep 2 versions, then we could keep both. 
2021-10-01 = Exports v1. Legacy exports. 
2023-07-01-preview = Exports v2. The new one with file partitioning, compression, file format, newer datasets etc... This is going to be the default experience going forward.